Automattic/wp-calypso
# Plans 2023 : Cosmetic Fixes Bundle

## Proposed Changes

- Reduce stroke size of tick in the plans comparison grid
- Increase stroke size of chevron down in the plans comparison grid smaller size plan selector view.

<img width="1334" alt="image" src="https://user-images.githubusercontent.com/3422709/218071244-e4ab56e9-537f-40b4-999f-f2fd18b0de64.png">

- Fixes alignment of downgrade button on `/plans` page - Automattic/martech#1496
<img width="1637" alt="image" src="https://user-images.githubusercontent.com/3422709/218318881-34131159-6cc6-47b5-aca0-d53482ebea58.png">

- Updated the Premium Themes image 

<img width="591" alt="image" src="https://user-images.githubusercontent.com/3422709/218318980-b6200ab7-3fba-48ab-b72c-add3eeedca1f.png">

- Removes brute protect from all screens.

## Testing Instructions



* Go through the `/start` onboarding flow and on the plans step, make sure the above aspects displays correctly
* Buy a business plan and go to the `/plans` page and make sure the above displays correctly, specially (Automattic/martech#1496) 

## Pre-merge Checklist




Fixes Automattic/martech#1493
Fixes Automattic/martech#1489
Fixes Automattic/martech#1481
Fixes Automattic/martech#1496
